src/psi/j/launchers/scripts/lib.sh

```diff
@@ -0,0 +1,42 @@
+#define _HASH_ #
+#define _BIN_BASH_ #!/bin/bash
+
```

Comment:
They are launched with bash explicitly. That said, _BIN_BASH_ is only useful at the beginning of lib.sh. Unfortunately, it does not work since (at least gcc) the preprocessor seems to insert a space before the #!/bin/bash. So this whole #include needs some work. But not right now.